plugins/catalog/api-report.md

```diff
@@ -64,12 +64,20 @@ export type BackstageOverrides = Overrides & {
   >;
 };
 
+// @public (undocumented)
+export type Breakpoint = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+
 // @public (undocumented)
 export const CatalogEntityPage: () => JSX.Element;
 
 // @public (undocumented)
 export const CatalogIndexPage: (props: DefaultCatalogPageProps) => JSX.Element;
 
+// @public (undocumented)
+export type CatalogInputPluginOptions = {
```

Comment:
The exported name of this should probably be `CatalogPluginOptions`

Reply:
That one's already taken; this is the input variant - https://github.com/backstage/backstage/blob/62e907393270103892d51dca2e959eb1b63ea9ab/plugins/catalog/src/plugin.ts#L76

Reply:
Yeah I know, it's internal vs external options. Maybe some completely different name could be better. This is all experimental though and it's a bit unfortunate to export. Perhaps slap an `@ignore` on it instead and leave it unexported?

Reply:
yep agreed, removed in 6c905ee